Cannot build: QFileOpenEvent does not name a type

### Operating System

Linux

### Version of Prism Launcher

latest git

### Version of Qt

Qt 5.15.8

### Description of bug

I get the following errors:
```
[ 72%] Building CXX object launcher/CMakeFiles/Launcher_logic.dir/ui/pages/global/LauncherPage.cpp.o
/home/david/.cache/kiss/proc/3576/build/prismlauncher/launcher/Application.cpp: In member function 'virtual bool Application::event(QEvent*)':
/home/david/.cache/kiss/proc/3576/build/prismlauncher/launcher/Application.cpp:917:31: error: 'QFileOpenEvent' does not name a type
  917 |         auto ev = static_cast<QFileOpenEvent*>(event);
      |                               ^~~~~~~~~~~~~~
/home/david/.cache/kiss/proc/3576/build/prismlauncher/launcher/Application.cpp:917:45: error: expected '>' before '*' token
  917 |         auto ev = static_cast<QFileOpenEvent*>(event);
      |                                             ^
/home/david/.cache/kiss/proc/3576/build/prismlauncher/launcher/Application.cpp:917:45: error: expected '(' before '*' token
  917 |         auto ev = static_cast<QFileOpenEvent*>(event);
      |                                             ^
      |                                             (
/home/david/.cache/kiss/proc/3576/build/prismlauncher/launcher/Application.cpp:917:46: error: expected primary-expression before '>' token
  917 |         auto ev = static_cast<QFileOpenEvent*>(event);
      |                                              ^
/home/david/.cache/kiss/proc/3576/build/prismlauncher/launcher/Application.cpp:917:54: error: expected ')' before ';' token
  917 |         auto ev = static_cast<QFileOpenEvent*>(event);
      |                                                      ^
      |                                                      )
/home/david/.cache/kiss/proc/3576/build/prismlauncher/launcher/Application.cpp:918:34: error: cannot convert '<brace-enclosed initializer list>' to 'QList<QUrl>'
  918 |         m_mainWindow->processURLs({ ev->url() });
      |         ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
In file included from /home/david/.cache/kiss/proc/3576/build/prismlauncher/launcher/Application.cpp:48:
/home/david/.cache/kiss/proc/3576/build/prismlauncher/launcher/ui/MainWindow.h:85:34: note:   initializing argument 1 of 'void MainWindow::processURLs(QList<QUrl>)'
   85 |     void processURLs(QList<QUrl> urls);
      |                      ~~~~~~~~~~~~^~~~
```

### Steps to reproduce

```bash
git clone https://github.com/PrismLauncher/PrismLauncher
cd PrismLauncher
git submodule update --init --recursive
cmake -S . -B build \
	-DCMAKE_BUILD_TYPE=Release \
	-DCMAKE_INSTALL_PREFIX="/usr" \
	-DENABLE_LTO=ON
cmake --build build -j$(nproc)
cmake --install build
```
...in reality I'm doing this inside of the KISS package manager, and the real script is as follows, but the above should be easier for you to replicate on anything else:
```sh
#!/bin/sh -e

export DESTDIR="$1"
git submodule update --init --recursive
cmake -S . -B build \
	-DCMAKE_BUILD_TYPE=Release \
	-DCMAKE_INSTALL_PREFIX="/usr" \
	-DENABLE_LTO=ON
cmake --build build -j$(nproc)
cmake --install build
```

### Suspected cause

`#include <QFileOpenEvent>` should be added to Application.cpp, as said in https://doc.qt.io/qt-5/qfileopenevent.html
If there is some way that it is supposed to be transitively imported, then that isn't working.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

--- a/launcher/Application.cpp
+++ b/launcher/Application.cpp
@@ -79,6 +79,7 @@
 #include <iostream>
 #include <mutex>
 
+#include <QFileOpenEvent>
 #include <QAccessible>
 #include <QCommandLineParser>
 #include <QDir>

